Lib/asyncio/queues.py

```diff
@@ -150,8 +184,18 @@ async def get(self):
         """Remove and return an item from the queue.
 
         If queue is empty, wait until an item is available.
+
+        Raise a QueueShutDown if _shutdown_state is set to
+        `SHUTDOWN_IMMEDIATE`.
+
+        Raise a QueueShutDown if _shutdown_state is set to
+        `SHUTDOWN` and queue is empty.
```

Comment:
```suggestion
        Raises QueueShutDown if the queue has been shut down and is empty, or
        if the queue has been shut down immediately.
```

Doc-strings are for humans